src/test/kotlin/graphql/kickstart/tools/FieldResolverScannerTest.kt

```diff
@@ -81,6 +81,16 @@ class FieldResolverScannerTest {
         assertEquals((meta as MethodFieldResolver).method.returnType, HullType::class.java)
     }
 
+    @Test
+    fun `scanner finds field resolver method using capitalize field_name`() {
+        val resolver = RootResolverInfo(listOf(CapitalizeQuery()), options)
+
+        val meta = scanner.findFieldResolver(FieldDefinition("id", TypeName("HullType")), resolver)
```

Comment:
Lets make sure the test will fail on all machines by setting the default locale to Turkish `Locale.forLanguageTag("tr-TR")` and resetting it back to default at the end. 
And add a comment to explain why we do that please.